### PR TITLE
Update User Guide & Icon

### PR DESCRIPTION
**Pull request by nyankoclaws**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -6,110 +6,131 @@
 
 # Dormie User Guide
 
-Dormie is a **desktop app for managing contacts, optimized for use via a  Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, Dormie can get your contact management tasks done faster than traditional GUI apps.
+Dormie is a **desktop app for managing contacts, optimized for use via a Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, Dormie can get your contact management tasks done faster than traditional GUI apps.
```
Comment (PallonCX) [inline docs/UserGuide.md:9]: Should CLI be "Command Line Interface" instead?

Comment (nyankoclaws) [reply docs/UserGuide.md]: updatedd

**Review by PallonCX — APPROVED** [review]
LGTM

**Review by donwong2308 — APPROVED** [review]
LGTM

**Review by larainezo — APPROVED** [review]
LGTM

**Review by JustWeiHao — APPROVED** [review]
LGTM

**Pull request merged**